components/webui_resources/webui_resources.cc

```diff
@@ -0,0 +1,805 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/components/webui_resources/webui_resources.h"
+
+#include "base/strings/utf_string_conversions.h"
+#include "brave/components/constants/url_constants.h"
+#include "brave/components/ipfs/buildflags/buildflags.h"
+#include "brave/components/tor/buildflags/buildflags.h"
+#include "build/build_config.h"
+#include "components/grit/brave_components_resources.h"
+#include "components/grit/brave_components_strings.h"
+#include "components/grit/components_resources.h"
+#include "ui/base/l10n/l10n_util.h"
+
+#if !BUILDFLAG(IS_ANDROID) && !BUILDFLAG(IS_IOS)
+#include "brave/components/brave_rewards/resources/grit/brave_rewards_resources.h"
+#endif
+
+namespace brave {
+
+const std::map<std::string, std::vector<WebUISimpleItem>>& GetWebUIResources() {
+  // clang-format off
```

Comment:
remove all format-disablers, add trailing commas everywhere and let clang-format do the job.